pkg/services/sqlstore/migrations/user_mig.go

```diff
@@ -151,6 +151,11 @@ func addUserMigrations(mg *Migrator) {
 		Postgres("UPDATE `user` SET uid='u' || lpad('' || id::text,9,'0') WHERE uid IS NULL;").
 		Mysql("UPDATE user SET uid=concat('u',lpad(id,9,'0')) WHERE uid IS NULL;"))
 
+	mg.AddMigration("Make sure default admin has a uid", NewRawSQLMigration("").
+		SQLite("UPDATE user SET uid=printf('u%09d',id) WHERE uid = '' AND id = 1 ;").
+		Postgres("UPDATE `user` SET uid='u' || lpad('' || id::text,9,'0') WHERE uid = '' AND id = 1;").
+		Mysql("UPDATE user SET uid=concat('u',lpad(id,9,'0')) WHERE uid = '' AND id = 1;"))
```

Comment:
Can we please add Spanner version too? (Using [`concat`](https://cloud.google.com/spanner/docs/reference/standard-sql/string_functions#concat) and [`lpad`](https://cloud.google.com/spanner/docs/reference/standard-sql/string_functions#lpad) should be fine)